pkg/spec/resources_transform.go

```diff
@@ -38,12 +36,12 @@ import (
 	_ "k8s.io/client-go/pkg/apis/extensions/install"
 )
 
-func getLabels(app *App) map[string]string {
-	labels := map[string]string{"app": app.Name}
+func (app *ControllerFields) getLabels() map[string]string {
+	labels := map[string]string{"deployment": app.Name}
```

Comment:
This will still be `app`, because of the following reasons:

- this is not tied to deployment in anyway
- user will still be considering it as app
- the error is user facing

Reply:
ack, changed